### PR TITLE
Update deps; add drawer state to store

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

**Comment by keanulee** [comment]
Oops broke the tabs. Fix in progress...

File: src/components/shop-app.js [hunk 1]
```diff
@@ -346,32 +344,33 @@ class ShopApp extends connect(store)(LitElement) {
...
     // Custom elements polyfill safe way to indicate an element has been upgraded.
     this.removeAttribute('unresolved');
   }
 
-  _stateChanged(state) {
-    const category = currentCategorySelector(state);
+  stateChanged(state) {
     this._page = state.app.page;
-    this._categories = state.categories;
+    this._categories = Object.keys(state.categories).map(key => state.categories[key]);
```
Comment (frankiefu) [inline src/components/shop-app.js]: Any reason to move `Object.keys(...)`here? This gets executed any time the store updates opposite of only when the properties that shop-app cares get update.

**Comment by keanulee** [comment]
I reset the commit history completely since the previous ones were just broken. Also tried to make the formatting of HTML template strings consistent:

```js
return html`
<h1></h1>
${condition ? html`
  <p></p>
` : condition2 ? html`
  <section></section>
` : null}`;
```

**Comment by frankiefu** [comment]
There is a strange behavior when running in Safari. When I clicked on the category tab (Men's Outerwear) the order of the tabs changed, like this:
screen shot 2018-10-11 at 11 55 59 am

**Comment by keanulee** [comment]
Looks like for Safari, `polymer serve` transforms object spread to a shim that uses `Object.defineProperty()`. This changes the ordering of keys, but only in Safari:

```js
> obj = {a:1,b:2,c:3}
< {a: 1, b: 2, c: 3}
> Object.defineProperty(obj, 'a', { value:2 })
< {b: 2, c: 3, a: 2}
```

Could use `Object.assign()` instead of spread. This isn't new to this PR though so wouldn't block on this.

**Pull request merged**